### ISSUE TITLE
Markdown template parser cannot decipher UTF-8 Unicode strings

### ISSUE THREAD

**Issue by nroach** [jhpyle/docassemble#21]
When uploading a UTF-8 encoded .md text file (during Wizard) the application accepts the upload but does not successfully parse the .md file.  Logs show an error `Error was:  'ascii' codec can't decode byte 0xe2 in position 83: ordinal not in range(128)`  Although the wizard appears to proceed, variables are not properly defined and running the interview errors out with a decode error.

This is based on the Dockerhub version current on the date this issue was submitted.

**Comment by nroach** [comment]
Additional Detail:

Error
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 96: ordinal not in range(128)
Log
```
Traceback (most recent call last):
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/flask/app.py", line 1612, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/flask/app.py", line 1598, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/docassemble/webapp/server.py", line 2713, in index
    interview = docassemble.base.interview_cache.get_interview(yaml_filename)
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/docassemble/base/interview_cache.py", line 20, in get_interview
    the_interview = interview_source.get_interview()
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/docassemble/base/parse.py", line 144, in get_interview
    return Interview(source=self)
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/docassemble/base/parse.py", line 2011, in __init__
    self.read_from(kwargs['source'])
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/docassemble/base/parse.py", line 2067, in read_from
    question = Question(document, self, source=source, package=source_package, source_code=source_code)
  File "/usr/share/docassemble/local/local/lib/python2.7/site-packages/docassemble/base/parse.py", line 1074, in __init__
    self.content = TextObject(definitions + unicode(data['content']), names_used=self.mako_names)
UnicodeDecodeError: 'ascii' codec can't decode byte 0x80 in position 96: ordinal not in range(128)
```

**Comment by jhpyle** [comment]
I'm sorry about that.  In some of my code I used the right mechanisms for loading Unicode files and in others I hadn't.  I have gone through the code and made all file-loading unicode-friendly.  I tried running the Wizard with a Markdown file containing unicode and it did not give me any problems.  The fixes are in the latest commit, which is building on Docker Hub now.  Thanks very much for reporting this!
